File: pgm/ABC_Visual_Basic/ABC_403_A.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim N As Integer
        N = Console.ReadLine()
        Dim A() As String
        A = Split(Console.ReadLine())
        Dim sum As Integer = 0
        For i As Integer = 0 To N - 1 Step 2
            sum += A(i)
        Next
        Console.WriteLine(sum)
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_403_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        Console.ReadLine();
        int[] A = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
        const int two = 2;
        const int zero = 0;
        Console.WriteLine(A.Where((a, i) => i % two == zero).Sum());
        Environment.Exit(0);
    }
}